src/replace/module.coffee

```diff
@@ -2,6 +2,9 @@ quibble = require('quibble')
 imitate = require('./imitate')
 isConstructor = require('./is-constructor')
 wrapWithConstructor = require('./wrap-with-constructor')
+_ = {
+  keys: require('lodash/keys')
```

Comment:
This ignores inherited properties. Not sure if this should be the desired functionality.